std/experimental/testing/should.d

```diff
@@ -0,0 +1,453 @@
+module std.experimental.testing.should;
+
+import std.exception;
+import std.conv;
+import std.algorithm;
+import std.traits;
+import std.range;
+
+public import std.experimental.testing.attrs;
+
+@safe:
+
+/**
+An exception to signal that a test case has failed.
+ */
+class UnitTestException: Exception
+{
+    this(in string[] msgLines, in string file, in ulong line)
+    {
+        import std.array;
+        super(msgLines.map!(a => getOutputPrefix(file, line) ~ a).join("\n"));
+    }
+
+private:
+
+    string getOutputPrefix(in string file, in ulong line) const
+    {
+        return "    " ~ file ~ ":" ~ line.to!string ~ " - ";
+    }
+}
+
+/**
+Verify that the condition is `true`.
+Throws: UnitTestException on failure.
+ */
+void shouldBeTrue(E)(lazy E condition,
+                  in string file = __FILE__, in ulong line = __LINE__)
+{
+    if(!condition) failEqual(condition, true, file, line);
```

Comment:
Nitpick. IMO having the expression on the same line as the condition is not phobos style